docs/Operators.md

```diff
@@ -19168,8 +19168,6 @@ data = np.array(
     dtype=np.float32,
 )
 reduced = np.maximum.reduce(data, axis=axes, keepdims=keepdims == 1)
-# print(reduced)
```

Comment:
Not sure about this ... may be this was intentional, to include the expected output in the documentation?

Reply:
> to include the expected output in the documentation

Yes that is the most likely reason... If it is really useful, we can still keep it on demand.